src/Raven.Server/Documents/TimeSeries/TimeSeriesStats.cs

```diff
@@ -419,10 +420,15 @@ public IEnumerable<Slice> GetTimeSeriesByPolicyFromStartDate(DocumentsOperationC
                     if (stats.Count == 0)
                         continue;
 
-                    DocumentsStorage.TableValueToSlice(context, (int)StatsColumns.Key, ref result.Result.Reader, out var slice);
                     var currentStart = new DateTime(Bits.SwapBytes(DocumentsStorage.TableValueToLong((int)StatsColumns.Start, ref result.Result.Reader)));
                     if (currentStart > start)
+                    {
+                        if (logger.IsOperationsEnabled)
```

Comment:
why operations?

Reply:
move all the added logs to be in information mode

Reply:
done